.github/workflows/docs-parity-updates.yml

```diff
@@ -14,19 +14,12 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout docs
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
```

Comment:
Since you already upgraded the checkout action version, you could get rid of the last two remaining deprecation warnings with:
- Line 36: `actions/setup-python@v4` (no changes other than the version bump necessary)
- Line 86: Using `GITHUB_OUTPUT` instead of `set-output`:
  - `echo "diff-count=$(cat resources/diff-check.log | wc -l)" >> $GITHUB_OUTPUT` should do the trick.
  - Here's an example: https://github.com/localstack/localstack-terraform-test/blob/main/.github/workflows/main.yml#L26
  - Here's a very short blog post on the deprecation (with examples): https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/